drivers/hmis_csv_twenty_twenty_four/app/models/hmis_csv_twenty_twenty_four.rb

```diff
@@ -0,0 +1,39 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisCsvTwentyTwentyFour
+  def self.table_name_prefix
+    'hmis_csv_twenty_twenty_four_'
+  end
+
+  def self.importable_files_map
+    {
+      'Export.csv' => 'Export',
+      'Organization.csv' => 'Organization',
+      'Project.csv' => 'Project',
+      'Client.csv' => 'Client',
+      'Disabilities.csv' => 'Disability',
+      'EmploymentEducation.csv' => 'EmploymentEducation',
+      'Enrollment.csv' => 'Enrollment',
+      'EnrollmentCoC.csv' => 'EnrollmentCoc',
```

Comment:
```suggestion
```

Reply:
Also I think HMISParticipation.csv and CEParticipation.csv are missing here.

Reply:
`importable_files_map` is part of the importer... I left it here since I was just going to have to put it back, but it is changed in the importer branch.